spec/generator_spec.rb

```diff
@@ -13,33 +13,37 @@
     end
   end
 
+  before(:all) do
```

Comment:
@deivid-rodriguez check this out - I think this makes more sense than what we were doing before. I noticed bundler was complaining when executing `bin/decidim` and using decidim's git repository *inside* the generated app (got somehow confused). This seems to do the trick!

Reply:
I think this is actually a bug in our generators, let me have a look!

Reply:
Yeah, somehow the `--path` and `--edge` options are having problems with latest `bundler` because the generator does not pick the right `Gemfile`. I'm pushing a fix now.